chainer/datasets/tox21.py

```diff
@@ -0,0 +1,150 @@
+import os
+import shutil
+import warnings
+import zipfile
+
+
+from chainer.dataset import download
+from chainer.datasets import tuple_dataset
+import numpy
+
+try:
+    from rdkit import Chem
+    from rdkit.Chem import rdMolDescriptors
+    _available = True
+except ImportError:
+    _available = False
+
+
+def check_available():
```

Comment:
This does not follow the convention of checking feature availability in Chainer.
How about simply `_available` variable public (i.e. remove `_` prefix), like `chainer.cuda.available`?